src/features/navigator/navigatorSlice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
Is there actually any state for this slice to contain? Or is it all managed by the URL?

Reply:
Find out in [the sequel](https://github.com/kbase/ui-refresh-test/pull/60)!